resume.json

```diff
@@ -0,0 +1 @@
+{"name":"Ryan","label":"Developer","picture":"N/A","email":"ryan.udugam@gmail.com","phone":"555-555-5555","website":"ryan.com","summary":"I like coding"}
```

Comment:
Is this a sample of what your outputted json looks like? Or was this for testing purposes?

Reply:
This was for testing, the structure is incorrect.